### PR TITLE
Changed names in committee

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: source/about/committee/index.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ The Committee is a 12 person team which is elected yearly to run all aspects of
...
-Treasurer: Ross O' Sullivan(sully)
+Secretary:  Oskar McDermott
+Treasurer: Ross O' Sullivan (sully)
 Events Officer: Cliodhna Harrison (thegirl)
-Public Relations Officer: Liam Rooney (ginger)
-System Administrators: Richard Walsh (koffee) & Lorcan Boyle (zergless) & TBD
-Webmaster: TBD
-Helpdesk: Cormac Smartt(pints) & TBD
-First Year Rep: TBD
+Public Relations Officer: Senan Kelly
```
Comment (butlerx) [inline source/about/committee/index.md]: username please

File: source/about/committee/index.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ The Committee is a 12 person team which is elected yearly to run all aspects of
...
+System Administrators: Richard Walsh (koffee) & Lorcan Boyle (zergless) & Cormac Smartt (pints)
+Webmaster: Cian Butle (butlerx)
+Helpdesk: Tom Doyle (greenday) & Dylan McDonald (grace)
+First Year Rep: George Burac
```
Comment (butlerx) [inline source/about/committee/index.md]: user name again

File: source/about/committee/index.md [hunk 3]
```diff
@@ -8,18 +8,18 @@ The Committee is a 12 person team which is elected yearly to run all aspects of
...
 
 ## Helpdesk
-The Helpdesk is made up of the friendly team of two Cormac Smartt(pints) & our
-second Education officer (Helpdesk) will be elected at an EGM to come.
+The Helpdesk is made up of the friendly team of two Tom Doyle(greenday) & our
+second Education officer Dylan McDonald(grace).
```
Comment (butlerx) [inline source/about/committee/index.md]: remove our second education officer

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ For developing new features and bug fixes, the `master` branch needs to be pulle
 and built upon. We follows a [Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration)
 model, where the master branch is always deployed to production.
 
-tep 2: Branch
+###Step 2: Branch
```
Comment (butlerx) [inline CONTRIBUTING.md]: rebase please

File: source/about/committee/index.md [hunk 5]
```diff
@@ -8,18 +8,18 @@ The Committee is a 12 person team which is elected yearly to run all aspects of
 the society
 
 Chairperson:  Noah O' Donnaile (cac)
-Secretary:  Cian Butler (butlerx)
-Treasurer: Ross O' Sullivan(sully)
+Secretary:  Oskar McDermott
```
Comment (butlerx) [inline source/about/committee/index.md]: also username

Comment (gruunday) [inline source/about/committee/index.md]: Do they have usernames yet?

Comment (butlerx) [inline source/about/committee/index.md]: Yes they would

File: source/about/committee/index.md [hunk 6]
```diff
@@ -8,18 +8,17 @@ The Committee is a 12 person team which is elected yearly to run all aspects of
...
+Treasurer: Ross O' Sullivan (sully)
 Events Officer: Cliodhna Harrison (thegirl)
-Public Relations Officer: Liam Rooney (ginger)
-System Administrators: Richard Walsh (koffee) & Lorcan Boyle (zergless) & TBD
-Webmaster: TBD
-Helpdesk: Cormac Smartt(pints) & TBD
-First Year Rep: TBD
+Public Relations Officer: Senan Kelly (sentriz)
+System Administrators: Richard Walsh (koffee) & Lorcan Boyle (zergless) & Cormac Smartt (pints)
+Webmaster: Cian Butle (butlerx)
```
Comment (gruunday) [inline source/about/committee/index.md]: my bad
fixed

Comment (butlerx) [inline source/about/committee/index.md]: my name is all that matters :stuck_out_tongue_closed_eyes:

**Pull request merged**